vertexai/internal/cmd/protoveneer/support/support.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Google LLC
```

Comment:
Is this the same file as the top-level support? I suppose the internal tool can use the package's support?

Reply:
Yes, the idea is that whoever is using the tool would copy this directory to where they needed it. Maybe someday it will have a stable import path.

For now I can delete this. The tool itself doesn't use it.